Gemfile

```diff
@@ -24,6 +26,7 @@ group :development, :test do
 end
 
 group :development do
+  gem 'factory_bot_rails', '~>6.2.0'
```

Comment:
if u use ffaker gem in prod - use factory too and u need it in test too

Reply:
fixed